### ISSUE TITLE
Found inconsistency in the following files - Backup Stuck

### ISSUE THREAD

**Issue by BethAr** [duplicati/duplicati#2912]
- [x] I have searched open and closed issues for duplicates.

## Environment info
- **Duplicati version**: 2.0.2.1_beta_2017-08-01
- **Operating system**: Manjaro Linux
- **Backend**: Jottacloud

## Description
A backup job stopped with an error (RequestStream time out, I believe) and now I'm unable to continue it. It fails with the following error:
```
Found inconsistency in the following files while validating database: ***********, actual size 633695364, dbsize 0, blocksetid: 19524 . Run repair to fix it.
```
(The *********** stands for the actual file name).

Repairing the database doesn't solve it, and purge-broken-files finds no error to purge. The backup is huge (512gb) and took a lot of time to upload. It failed on the last 10gb I believe. I don't know what to do.

## Steps to reproduce
I don't know how to reproduce it, I wasn't doing anything in particular when it failed.

- **Actual result**:
The backup is, as far as I can tell, lost. Even though I can repair from it, I have no way of continuing it, it always fails immediately.

- **Expected result**:
I should be able to continue my backup. Either purge-broken-files should fix the problematic file or the database repair should correct the inconsistency.

**Comment by BethAr** [comment]
It happened again today, with another backup. The error that interrupts the upload is:
```
GetRequestStream timed out
```
After this, I can no longer continue the backup, I always get the `Found inconsistency` bug. Repairing the database doesn't work, purge-broken-files doesn't work. I'm at a loss.

**Comment by BethAr** [comment]
Ok, I managed to fix the issue. Here is what I did:
- The `Found Inconsistency` error requires that the database be **recreated**, repairing is not enough.
- The `GetRequestStream timed out` error happens because it takes too long to list all the files, and the `http-operation-timeout` configuration needs to be set higher.
  - I set mine to 30 minutes and the time out stopped.
  - The option is not available in the Beta version. I updated to `2.0.2.13_canary_2017-11-22`


Thanks to **JonMikeIV** and **tophee**, on the Duplicati forums, for walking me through.
